metricbeat/module/docker/helper.go

```diff
@@ -16,13 +16,29 @@ type Container struct {
 	Socket *string
 }
 
-func InitCurrentContainer(container *docker.APIContainers) *Container {
+func (c *Container) ToMapStr() common.MapStr {
+	m := common.MapStr{
+		"id":   c.Id,
+		"name": c.Name,
+		// TODO: Is this really needed
+		"socket": GetSocket(),
```

Comment:
The main objectif of having it as a field is to have an clear idea about the conf and to check it as well.  Espacially in case we have two docker modules running and each one listen on different socket. Since  we can bind docker to another host or port using a tcp connection, it's always good to know if it listen on TCP or a Unix socket or both, that's why it's present n all the metricsets!